docker-testing/in-container-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e
```

It does not catch everything but it catches a lot.

Reply:
`set -e` is not incompatible with manual error handling at all. It would be totally unusable otherwise. `set -e` catches many failures that the developer did not anticipate or was too lazy or too optimistic to handle manually - now and in the future.

The only problem with `set -e` is the false, 100% safety impression that it can give to some inexperienced people. For everything else it's fine.

https://github.com/thesofproject/sof-test/issues/312